source/_integrations/guardian.markdown

```diff
@@ -31,7 +33,8 @@ There is currently support for the following device types within Home Assistant:
 - **Binary sensor**: reports the status of the onboard leak detector and access point
 - **Button**: add various configuration controls
 - **Sensor**: reports on the device's detected temperature and uptime
-- **Switch**: allows the user to open and close the valve
+- **Switch**: allows the user to enable and disable the onboard access point
+- **Valve**: allows the user to open and close the valve (deprecated)
```

Comment:
I think the (deprecated) comment is on the wrong place?

Reply:
Ah, yes. Based on your comments in https://github.com/home-assistant/core/pull/107423, should I mention deprecation at all?

Reply:
Probably too soon so I'd take it out for now